Makefile

```diff
@@ -3,72 +3,91 @@
 ifdef MODULE
```

Comment:
You have to also adjust the `make-targets` accordingly so you can call the correct targets from the subdirectories